examples/0-counter-part.elm

```diff
@@ -47,7 +45,7 @@ update msg model =
       init
 
     CounterMsg msg' -> 
-      Parts.update CounterMsg msg' model
+      Counters.pass CounterMsg msg' model
```

Comment:
This is the part where I'm thinking that every new component will require the user to put in an extra clause in `update`: Suppose I also have a `Button` component. I'd need a boilerplate clause

    ButtonMsg msg' -> 
      Buttons.pass ButtonMsg msg' model 

Right? For UI libraries, I might be using Buttons, Checkbox, Toggles, Tooltips, Tabs, Sliders... Lots of boilerplate.